main.go

```diff
@@ -72,6 +72,9 @@ func main() {
 
 	// Add the needed Gateway intents.
 	s.AddIntents(gateway.IntentGuildMessages)
+	s.AddIntents(gateway.IntentMessageContent)
+	s.AddIntents(gateway.IntentDirectMessages)
+	s.AddIntents(gateway.IntentGuildMessages)
```

Comment:
Added twice.

Reply:
Good catch; thanks for the review! Addressed in <https://github.com/acmcsufoss/sendlimiter/pull/2/commits/45c33f9a01bb1893c644ca852d56241ec720913c>.